spec/models/card_spec.rb

```diff
@@ -0,0 +1,83 @@
+require "rails_helper"
+
+describe  do
+  it "is valide with original_text, translated_text" do
+    expect(build(:card, original_text: "Стул", translated_text: 'Chair')).to be_valid
```

Comment:
Line is too long. [85/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.